docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,95 @@
+[[indicators-of-compromise]]
+= Indicators of compromise
+
+The Indicators page collects data from enabled threat intelligence feeds and provides a centralized view of indicators, also known as indicators of compromise (IoCs). This topic helps you set up the Indicators page  and explains how to work with IoCs.
+
+[role="screenshot"]
+image::images/indicators-table.png[Shows the Indicators page]
+
+.Requirements
+[sidebar]
+--
+You must have _one_ of the following installed on the hosts you want to monitor:
+
+* *{agent}* - Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] and ensure the agent's {fleet-guide}/view-elastic-agent-status.html[status] is `Healthy`. Refer to {fleet-guide}/fleet-troubleshooting.html[{fleet} Troubleshooting] if it isn't.
+* *{filebeat}* - Install {filebeat-ref}/filebeat-installation-configuration.html[{filebeat}] version 8.x. Earlier {filebeat} versions are incompatible with ECS and will prevent indicator data from displaying in the Indicators table.
+--
+
+[discrete]
+[[ti-indicators]]
+== Threat intelligence and indicators
+Threat intelligence is a research function that helps analyze current and emerging threats, and recommends appropriate actions to strengthen a company's security posture. Threat intelligence requires proactivity to be useful, such as gathering, analyzing, and investigating various threat and vulnerability data sources.
+
+An indicator, also referred to as an indicator of compromise (IoC), is a piece of information associated with a known threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses, and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
+
+[discrete]
+[[setup-indicators-page]]
+== Set up the Indicators page
+
+Install a threat intelligence integration to add indicators to the Indicators page.
+
+NOTE: If you've already installed a threat intelligence integration, you don't need to complete the following steps. Indicator data will automatically display on the Indicators page.
```

Comment:
I might be missing what this note is trying to do. The previous sentence says "install this," and then this note says "if you've already installed this, then you don't need to do the installation steps." 

That seems very obvious; if you've already installed something, then of course you can ignore the installation instructions. Is there more nuance that I'm missing?

Reply:
Hm, good point. The lead-in sentence already says that threat intel integrations need to be installed for the page to display data. So if users have already installed those, they can likely conclude that they've completed the setup steps. Maybe this note isn't needed after all.